Line/server/app.js

```diff
@@ -0,0 +1,147 @@
+// Copyright 2015-2016, Google, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the 'License');
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an 'AS IS' BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START app]
+'use strict';
+
+var firebase = require('firebase');
```

Comment:
Fixed in this [commit](https://github.com/firebase/custom-auth-samples/pull/1/commits/c4adf034a54e67b5f04aaa1b4ae7a8098cb668bc) :)